### PR TITLE
feat(pose_initilizer): set intial pose directly

### PR DESCRIPTION
**Pull request by YamatoAndo**
## Description

see https://github.com/autowarefoundation/autoware.universe/pull/6692

## Related links

see https://github.com/autowarefoundation/autoware.universe/pull/6692

## Tests performed

see https://github.com/autowarefoundation/autoware.universe/pull/6692

## Notes for reviewers

see https://github.com/autowarefoundation/autoware.universe/pull/6692

## Interface changes

see https://github.com/autowarefoundation/autoware.universe/pull/6692

## Effects on system behavior

see https://github.com/autowarefoundation/autoware.universe/pull/6692

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: autoware_launch/config/localization/pose_initializer.param.yaml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 /**:
   ros__parameters:
+    initial_pose_directly:
```
[the remark this answers is not in the document]
Comment (YamatoAndo) [reply autoware_launch/config/localization/pose_initializer.param.yaml]: sorry, thanks!

[the remark this answers is not in the document]
Comment (YamatoAndo) [reply autoware_launch/config/localization/pose_initializer.param.yaml]: https://github.com/autowarefoundation/autoware_launch/pull/937/commits/2e6340cd48f6fe1e8292a1579896f8775789f9d6

**Review by TaikiYamada4 — APPROVED** [review]
LGTM!

**Review by SakodaShintaro — APPROVED** [review]
Looks Good To Me

**Pull request merged**